index.html

```diff
@@ -14,6 +14,373 @@
   <title>Itumeleng Rantsane | Portfolio </title>
 </head>
 
+<<<<<<< HEAD
+  <body>
+    <div class="hero">
+      <header class="header bg-white">
+        <nav class="navbar flex">
+          <a href="#" class="navbar__link link fz-20 fw-700 lh-20">I.R</a>
+          <div class="navbar__menu">
+            <ul class="navbar__menu__ul ul flex">
+              <li class="navbar__menu__li">
+                <a
+                  href="#works"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Portfolio</a
+                >
+              </li>
+              <li class="navbar__menu__li">
+                <a
+                  href="#about"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >About</a
+                >
+              </li>
+              <li class="navbar__menu__li ">
+                <a
+                  href="#contact"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Contact</a
+                >
+              </li>
+            </ul>
+          </div>
+          <img
+            class="navbar__hamb "
+            src="images/hamburger.png"
+            alt="hamburger icon"
+          />
+        </nav>
```

Comment:
- [ ] When I close the mobile menu, The hamburger icon is broken, Please fix this issue.

![image](https://user-images.githubusercontent.com/83803414/203592977-243d4614-4d5e-4943-a459-7758e671550d.png)